src/builder/builder.js

```diff
@@ -62,10 +62,7 @@ module.exports = function (createChunker, ipldResolver, createReducer, _options)
     waterfall([
       (cb) => DAGNode.create(d.marshal(), cb),
       (node, cb) => {
-        ipldResolver.put({
-          node: node,
-          cid: new CID(node.multihash)
-        }, (err) => cb(err, node))
+        ipldResolver.put(node, new CID(node.multihash), (err) => cb(err, node))
```

Comment:
- readability before: 8
- readability after: 2

this comes from two things

1. no named parameters, before the naming made clear what belonged where and as what it was used
2. `put(node, cid, cb)` is a confusing signature, as `put` is closes associated with something like a key value store, where you would have `put(key, value, cb)`. But here you first pass the node, then an id.